bin/ecompress

```diff
@@ -71,9 +71,9 @@ while [[ $# -gt 0 ]] ; do
 				# note: to save time, we need to do this only if there's
 				# at least one compressed file
 				case ${path} in
-					*.Z|*.gz|*.bz2|*.lzma|*.xz)
+					*.Z|*.gz|*.bz2|*.lzma|*.xz|*.zst)
```

Comment:
Don't we want `.lz4`, `.lz`, `lzo` here as well?

Reply:
Good point, I'll add that now.